package.json

```
@@ -102,5 +102,8 @@
     "webpack-cli": "^4.5.0",
     "webpack-dev-server": "^3.11.2"
   },
-  "dependencies": {}
+  "dependencies": {
```

```suggestion
  "dependencies": {
```

Please put this under dev dependencies

---

Updated now to keep in under devDependencies